docs/docs/40-operator-guide/40-security/30-access-controls.md

```diff
@@ -85,21 +83,46 @@ In the following example, the `ServiceAccount` resource is mapped to all of:
 * All users with a `groups` claim  containing _either_ the `devops` or
   `kargo-admin` group.
 
+### Map example
+
 ```yaml
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: admin
   namespace: kargo-demo
   annotations:
-    rbac.kargo.akuity.io/claim.sub: alice,bob
-    rbac.kargo.akuity.io/claim.email: carl@example.com
-    rbac.kargo.akuity.io/claim.groups: devops,kargo-admin
+    rbac.kargo.akuity.io/claims:
+      sub: 
+        - alice
+        - bob
+      email: carl@example.com
+      groups: 
+        - devops
+        - kargo-admin
+      "special:key": value
```

Comment:
This isn't a valid annotation. The value should be a _string representation_ of a JSON object.